src/aat/resources/features/F-032/F-032.feature

```diff
@@ -0,0 +1,57 @@
+@F-032
+Feature: F-032: Get printable documents for a case
+
```

Comment:
Feature name should : Get printable documents (from excel)

Reply:
an earlier review comment said to elaborate the name from the excel sheet 🤷‍♂️

Reply:
It seems like we shouldn't be changing the feature name given in excel, so i am updating that as per excel now.